### ISSUE TITLE
Allow people to convert steps to paths as an other answer

### ISSUE THREAD

**Issue by peternewman** [streetcomplete/StreetComplete#5217]
**Use case**
Off the back of the very useful #3288 it would be nice if there was a quick way to stop all the step related tags being asked and let you get on with the next relevant ones (e.g. lit or surface). Just adding a note would also stop those being answered as it blocks all quests until the note is resolved.

**Proposed Solution**
Just offer the reverse of https://github.com/streetcomplete/StreetComplete/issues/3288

Removing at least all the step related quests SC adds

**Comment by thany** [comment]
I've had more than one instance already where stairs had been converted to a slope. I would either have to leave a note, and basically let someone else fix it, or go home and fix it myself before returning to map any new slope-related quests.

**Comment by peternewman** [comment]
Yeah I've seen stairs to slopes which triggered generating this issue. But also micro-mapping landings and similar on long staircases.

**Comment by westnordost** [comment]
Hm well, the usefulness of this is undoubtedly lower as the other way round, as usually I expect this only serves to remove some part of the steps or add e.g. an intermediate platform that is for some meters not some steps. I.e. mostly for micromapping.
The other way round is more important because when there are steps, it is an actual barrier.

**Comment by matkoniecz** [comment]
Also, on moving to not steps it is necessary to remove some tags (and some, like handrail maybe should be removed maybe not)

**Comment by thany** [comment]
> The other way round is more important because when there are steps, it is an actual barrier.

This is true, but a set of stairs converted to a slope might also be a barrier. For example, to someone who is not so able-bodies but can still walk, and the slope is just about too steep for them, where a set of stairs would have been fine.

Either way, just as adding steps involves little more than adding some tags to a (piece of a) path, converting stairs to a slope involves little more than the reverse: removing a set of tags pertaining to stairs will make it a slope.

Unless we're also supposed to map the gradient of slopes, which is probably difficult without a sign.

**Comment by peternewman** [comment]
> The other way round is more important because when there are steps, it is an actual barrier.

The particular example that triggered me to raise this issue, they'd removed all the steps, I'm not sure they'd necessarily made it a shallow enough gradient for a wheelchair user to get up unassisted but for a pushchair/pram or a bike its now just a steep ramp, so much easier than stairs.

> Also, on moving to not steps it is necessary to remove some tags (and some, like handrail maybe should be removed maybe not)

Yeah, well is there a way to tag a ramp of some sort (as distinct from just a path on a gradient), if so then ideally I guess we'd remove all the things like handrails @matkoniecz and then have new quests to survey if there is a handrail on this ramp etc.

**Comment by danieldegroot2** [comment]
> Yeah I've seen stairs to slopes which triggered generating this issue. But also micro-mapping landings and similar on long staircases.

> Sometimes the steps are in the wrong section and would cause wheelchair routing to go the wrong way.

sidenote: There are cases of regular gate in the middle of steps, with or without clear landing.
